problem_builder/mentoring.py

```diff
@@ -239,6 +239,7 @@ class MentoringBlock(BaseMentoringBlock, StudioContainerWithNestedXBlocksMixin,
     ok to continue.
     """
     # Content
+    USER_STATE_FIELDS = ['active_step', 'num_attempts']
```

Comment:
`active_step` should not be exposed here - only on `MentoringWithExplicitStepsBlock`. Use of `MentoringBlock` itself with steps (mode = 'assessment') is deprecated and we don't need to support it.